components/Hero/StoryHero.tsx

```diff
@@ -1,40 +1,54 @@
 import React from 'react';
+import Image from 'next/image';
 import { cnb } from 'cnbuilder';
 import { Container } from '../Container';
+import { CtaLink } from '../Cta';
 import { Grid } from '../Grid';
+import { FlexBox } from '../FlexBox';
 import { Heading, Text, Paragraph } from '../Typography';
-import { paletteAccentColors, PaletteAccentColorType } from '../../utilities/colorPalettePlugin';
-import { accentBorderColors } from '../../utilities/datasource';
-import { getProcessedImage } from '../../utilities/getProcessedImage';
+import { paletteAccentColors, PaletteAccentColorType } from '@/utilities/colorPalettePlugin';
+import { accentBorderColors } from '@/utilities/datasource';
+import { getProcessedImage } from '@/utilities/getProcessedImage';
+import { slugify } from '@/utilities/slugify';
 import { SbImageType } from '../Storyblok/Storyblok.types';
 import * as styles from './StoryHero.styles';
 
 export type StoryHeroProps = {
   title: string;
   byline?: string;
-  intro?: string;
   publishedDate?: string;
+  dek?: string;
   heroImage?: SbImageType;
+  aspectRatio?: styles.ImageCropType;
+  mobileImage?: SbImageType;
+  mobileAspectRatio?: styles.ImageCropType;
+  alt?: string;
+  caption?: string;
   isVerticalHero?: boolean;
   isLeftImage?: boolean;
   isLightHero?: boolean;
   tabColor?: {
     value?: PaletteAccentColorType;
   }
-  taxonomy?: string[];
+  topics?: string[];
 };
 
 export const StoryHero = ({
   title,
   byline,
-  intro,
+  dek,
   publishedDate,
   heroImage: { filename, focus } = {},
+  mobileImage: { filename: mobileFilename, focus: mobileFocus } = {},
+  alt,
```

Comment:
Instead of using the `alt` field that comes with the Storyblok image component, I choose to add my own alt field in Storyblok so it's not hidden inside the image edit screen. We need an alt for the Story hero image, so having it exposed in Storyblok makes it clearer to the author that they need to enter one.